lib/sidekiq/statistic/views/styles/sidekiq-statistic-dark.css

```
@@ -0,0 +1,126 @@
+/* === DARK MODE STYLES === */
+
+/* === CONTAINER === */
+
+.statistic__container {
+  background-color: #282828;
+}
+
+/* === DATEPICKER === */
+
+.statistic__datepicker {
+  border-bottom: 1px solid #333333;
+}
+
+.statistic__datepicker input {
+  background: #333333;
+  border: 1px solid #555;
+  box-shadow: 0 0 5px rgba(255, 255, 255, 0.1);
+}
+
+#statistic__search {
+  background: #333333;
+  border: 1px solid #555;
+  box-shadow: 0 0 5px rgba(255, 255, 255, 0.1);
+  -webkit-border-radius: 4px;
+  -moz-border-radius: 4px;
+  border-radius: 4px;
+}
+
+/* === UI DATEPICKER jQuery === */
+
+.ui-datepicker {
+  background-color: #282828;
+  border: 1px solid #d5d5d5;
+}
+
+.ui-datepicker-today {
+  background: #CD3844;
+}
+
+.ui-datepicker a:active, a:hover, a:visited {
```

You need to add the entire selector after each `,`, otherwise It will be applied to every `a` tag selector. and only the first will be scopped to the `ui-datepicker`
```suggestion
.ui-datepicker a:active, 
.ui-datepicker a:hover, 
.ui-datepicker a:visited {
```